src/server/chromium/crPage.ts

```diff
@@ -969,7 +969,7 @@ class FrameSession {
     });
     const frameId = nodeInfo && typeof nodeInfo.node.frameId === 'string' ?
       nodeInfo.node.frameId : null;
-    documentElement.dispose();
+    void documentElement.dispose();
```

Comment:
Same here, we can make handle.dispose() not return a Promise.

Reply:
> Overall, I think this adds more noise than solves problems. Some are good catches though. I'd advise against enabling the plugin.

Yeah, I'm not fond of this `void` prefixes all over the code. Without them there is a lot of noise in the linter output masking real issues but we can live with that for now.

Reply:
Changed `dispose()` to not return promise.